installing/installing_bare_metal/upi/installing-bare-metal-network-customizations.adoc

```diff
@@ -0,0 +1,223 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-bare-metal-network-customizations"]
+= Installing a user-provisioned bare metal cluster with network customizations
+include::_attributes/common-attributes.adoc[]
+:context: installing-bare-metal-network-customizations
+
+toc::[]
+
+In {product-title} {product-version}, you can install a cluster on bare
+metal infrastructure that you provision with customized network configuration
+options. By customizing your network configuration, your cluster can coexist
+with existing IP address allocations in your environment and integrate with
+existing MTU and VXLAN configurations.
+
+When you customize {product-title} networking, you must set most of the network configuration parameters during installation. You can modify only `kubeProxy` network configuration parameters in a running
+cluster.
+
+== Prerequisites
+
+* You reviewed details about the xref:../../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../../installing/overview/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* If you use a firewall and plan to use the Telemetry service, you xref:../../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured the firewall to allow the sites] that your cluster requires access to.
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../../../installing/installing_bare_metal/upi/installing-restricted-networks-bare-metal.adoc#installing-restricted-networks-bare-metal[Installing a user-provisioned bare metal cluster on a restricted network] for more information about performing a restricted network installation on bare metal infrastructure that you provision.
+
+[id="installation-requirements-user-infra_{context}"]
+== Requirements for a cluster with user-provisioned infrastructure
+
+For a cluster that contains user-provisioned infrastructure, you must deploy all
+of the required machines.
+
+This section describes the requirements for deploying {product-title} on user-provisioned infrastructure.
```

Comment:
I wonder if this line should go first in the section, as the current first line is one of those very requirements.